Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-3: The Trial of all Crimes, except in Cases of Impeachment, shall be by Jury; and such Trial shall be held in the State where the said Crimes shall have been committed; but when not committed within any State, the Trial shall be at such Place or Places as the Congress may by Law have directed.
+3: The supreme Court shall have the authority, with the assent of two-thirds of the Justices, to decline to hear a case over which they have appellate jurisdiction, or with unanimous assent, to decline to hear a case over which they have original jurisdiction; but if the Chief Executives, Attorneys General, or Legislative bodies of more than one-third of the States shall be Plaintiff to a case, the supreme Court shall hear the case.
+
+4: The Trial of all Crimes, except in Cases of Impeachment, shall be by Jury; and such Trial shall be held in the State where the said Crimes shall have been committed; but when not committed within any State, the Trial shall be at such Place or Places as the Congress may by Law have directed.
 
 Section 3
-1: Treason against the United States, shall consist only in levying War against them, or in adhering to their Enemies, giving them Aid and Comfort. No Person shall be convicted of Treason unless on the Testimony of two Witnesses to the same overt Act, or on Confession in open Court.
+1: Treason against these United States, shall consist only in levying War against them, or in adhering to their Enemies, giving them material Aid and Comfort. No Person shall be convicted of Treason unless on the Testimony of two Witnesses to the same overt Act, or on Confession in open Court. In the event that a State should secede from the Union, no trial of Treason shall be brought, unless the seceding State shall have declared War against these United States.
 
 2: The Congress shall have Power to declare the Punishment of Treason, but no Attainder of Treason shall work Corruption of Blood, or Forfeiture except during the Life of the Person attainted.
 
 Article IV (Article 4 - States' Relations)
 Section 1
-Full Faith and Credit shall be given in each State to the public Acts, Records, and judicial Proceedings of every other State. And the Congress may by general Laws prescribe the Manner in which such Acts, Records and Proceedings shall be proved, and the Effect thereof.
+1: Full Faith and Credit shall be given in each State to the public Acts, Records, and judicial Proceedings of every other State. And the Congress may by general Laws prescribe the Manner in which such Acts, Records and Proceedings shall be proved, and the Effect thereof.
+
+2: The Government of each State shall have the right to challenge the Government of any other state to prove its Acts, Records, and Proceedings, and their compliance with this Constitution and with the Constitution of the state. When such a challenge is issued, the supreme Court shall have original jurisdiction.
```

Comment:
Another specific callout to Texas v. Pennsylvania. It remains unfortunate that this needs to be spelled out, but given the decision by 7 of those 9 justices in that case, it apparently must be.